web/src/frontend/src/digisos/skjema/kontaktinfo/index.tsx

```diff
@@ -19,6 +19,7 @@ class Kontaktinfo extends React.Component<{}, {}> {
 					<InputFaktum
 						required={true}
 						faktumKey="kontakt.kontonummer"
+						className="kontakt___kontonummer"
```

Comment:
Har vi mulighet for å koble nightwatch til komponenter uten å bruke klassenavn? Litt ugreit å ha klasser som ikke har noe styling. Men ikke veldig viktig.

Reply:
nightwatch fungerer med vanlig css-selektorregler. For andre nightwatchtester så har vi prøvd så gått det lar seg gjøre å kun selekte på elementnivå og der det ikke har gått har vi lagt til klasser med prefiks js-test-.... I foreldrepengertestene brukte vi mye faktumKey/value. Ta gjerne en titt der Thomas :)

Reply:
Disse to elementene har jo id, så da er det nok bedre å bruke den.

Reply:
Klassen ble lagt på fordi nightwatch og google chrome ikke klarte å finne elementene når id ble bruk som selector.

Ingen av disse selectorene ser ut til å fungere:  
```
  kontonummer: '#kontakt.kontonummer'
  kontonummer: '#kontakt\.kontonummer'
  kontonummer: 'input[id=kontakt.kontonummer]',
  kontonummer: 'input[id=kontakt\.kontonummer]'
```

Reply:
Muligens det ikke er bra å ha punktum i id. Vi kan se på å endre dette senere, slik at vi erstatter punktum med - eller noe

Reply:
`#kontakt.kontonummer` betyr jo noe med id `#kontakt` og klasse `kontonummer`, så dumt om idene inneholder punktum.

Tror escaping burde virke, men du må nok escape slashen, altså dobbel slash `kontonummer: '#kontakt\\.kontonummer'`

Reply:
Jeg kan lage en enkel PR som endrer ID-genereringen